test/cpp/test_aten_xla_tensor_4.cpp

```diff
@@ -1319,6 +1319,27 @@ TEST_F(AtenXlaTensorTest, TestSplitWithSizes) {
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestSplitDropRemainder) {
```

Comment:
Is this a missing test case?

Reply:
Oh I see, it's testing drop_remainder = true. So we add support for drop_remainder=true in this PR

Reply:
You got it!